plugins/events-backend-module-github/api-report.md

```diff
@@ -4,9 +4,14 @@
 
 ```ts
 import { BackendFeature } from '@backstage/backend-plugin-api';
+import { Config } from '@backstage/config';
 import { EventParams } from '@backstage/plugin-events-node';
+import { RequestValidator } from '@backstage/plugin-events-node';
 import { SubTopicEventRouter } from '@backstage/plugin-events-node';
 
+// @public
+export function createGithubSignatureVerifier(config: Config): RequestValidator;
```

Comment:
Would it make sense to use the nomenclature "Validator" instead of "Verifier" in the name to sync it up with the type? They are pretty equivalent as words.